ui/main/src/app/modules/admin/components/table/admin-table.directive.ts

```diff
@@ -201,18 +201,9 @@ export abstract class AdminTableDirective implements OnDestroy {
             columnDefs[index] = columnDef;
         });
 
-        const deleteActionCellClassRules = {
-            'action-cell-delete-admin': (params) =>
```

Comment:
It is possible to remove the css class "action-cell-delete-admin" from styles.scss as it not used anymore